pkg/image/resolve.go

```diff
@@ -222,90 +80,35 @@ func GetImages(systemChartPath, chartPath string, externalImages map[string][]st
 	return imagesList, imagesAndSourcesList, nil
 }
 
-func setImages(source string, imagesFromArgs []string, imagesSet map[string]map[string]bool) {
-	for _, image := range imagesFromArgs {
-		addSourceToImage(imagesSet, image, source)
-	}
-}
-
-func convertMirroredImages(imagesSet map[string]map[string]bool) {
-	for image := range imagesSet {
-		convertedImage := img.Mirror(image)
-		if image == convertedImage {
-			continue
-		}
-		for source, val := range imagesSet[image] {
-			if !val {
-				continue
-			}
-			addSourceToImage(imagesSet, convertedImage, source)
-		}
-		delete(imagesSet, image)
-	}
-}
+func CreateCatalogImageListConfigMap(cm *v1.ConfigMap, catalog *v3.Catalog) (err error) {
```

Comment:
As in how this new min/max constraint based logic performs compared to the old logic when used by this controller? How would I go about testing that? (Just in case, this function was not added in this PR, just moved when I broke down resolve.go)